homeassistant/components/broadlink/device.py

```diff
@@ -0,0 +1,57 @@
+"""Support for Broadlink devices."""
+from functools import partial
+import logging
+
+from broadlink.exceptions import (
+    AuthorizationError,
+    BroadlinkException,
+    ConnectionClosedError,
+    DeviceOfflineError,
+)
+
+from .const import DEFAULT_RETRY
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BroadlinkDevice:
+    """Manages a Broadlink device."""
```

Comment:
In my next PR I will implement config entries and this will become a singleton. The device will be responsible for initializing the platforms, and not the other way around.